app/src/main/java/com/criteo/testapp/listener/TestAppNativeAdListener.java

```diff
@@ -28,21 +28,15 @@
 
   private final String tag;
   private final String prefix;
-  private final ViewGroup adLayout;
 
-  public TestAppNativeAdListener(String tag, String prefix, ViewGroup adLayout) {
+  public TestAppNativeAdListener(String tag, String prefix) {
     this.tag = tag;
     this.prefix = prefix;
-    this.adLayout = adLayout;
   }
 
   @Override
   public void onAdReceived(@NonNull CriteoNativeAd nativeAd) {
     Log.d(tag, prefix + " - Native onAdReceived");
-
-    View view = nativeAd.createNativeRenderedView(adLayout.getContext(), adLayout);
```

Comment:
By removing this, the advanced native is broken. Nothing is shown in the activity.